lib/notification-issue.coffee

```diff
@@ -190,8 +196,9 @@ class NotificationIssue
           return packName unless /^\.\./.test(relativePath)
       @getPackageNameFromFilePath(filePath)
 
-    if options.detail? and packageName = getPackageName(options.detail)
-      return packageName
+    if options.detail?
+      packageName = getPackageName(options.detail)
+      return packageName if packageName?
```

Comment:
Oops, I forgot to fully revert it after I moved the regex out of this if.  Fixing now...